### DISCUSSION TITLE
No power meter data anymore

### DISCUSSION THREAD

**Discussion by Farquarl** [hoylabs/OpenDTU-OnBattery#1001] [Q&A]
Hey folks,

since a couple of days i don't get any data from my power meter anymore. Its weird since it worked before and i don't know whats wrong.

Im using a Hitchi IR reader running tasmota to get the data via JSON.
If i enter the Tasmota webUI it gives the right readings. I can also see the readings under the JSON path. However under the settings "power meter" whenever i run a test it returns a "HTTP Error connection refused"
As this is the first time i come in contact with stuff like JSON etc. im out of my water. Can anyone lead me to the solution? Anything i need to check?
I check that both the DTU and Tasmota are connected to the same WIFI.

![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/35021168/d641e99c-816c-466d-96d9-78673bd6829d)
![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/35021168/fd3544a4-2535-4c11-bb20-616b4fd2bdc0)
![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/35021168/006c86c0-9e07-426f-8586-d96ffabef028)

**Comment by rychom** [comment]
You can try different things:
reboot tasmota and dtu
check if the IP is still the right one, maybe it's just cached stuff...
maybe an invisible character in the url (space e.g?), or problems with "%20", try "+" instead
but all in all... it seems to be correct...

**Reply by Farquarl** [reply]
Ahh shoot, 

- I rebooted both devices
- I checked the IP
- If i take the same link from the DTU and copy it into my browser it shows the right values...

Maybe i should completely reflash and reset the config to make it work again.
Right now i just feed in 300W constant for the baseload of my house.

**Reply by schlimmchen** [reply]
> I check that both the DTU and Tasmota are connected to the same WIFI.

Have you (recently) disabled the function "allow communication between Wi-Fi clients" or installed an update on your Wi-Fi Access Point?

Does the Tasmota device only speak HTTPS? (not sure if they speak HTTPS at all)

**Reply by helmutkohlkanzler** [reply]
Hi, I have got exactly the same issue and tested things the same way. @Farquari did you find a solution? BTW, the tasmota device speaks http.

**Reply by helmutkohlkanzler** [reply]
... and yes, wifi clients are allowed to communicate...

**Reply by Farquarl** [reply]
Hey,
So my problem was that I have two routers one is set up as a repeater. For whatever reason the Hitch and the DTU need to be connected to the same router to work.

**Reply by helmutkohlkanzler** [reply]
Problem solved: while my dns works fine for many devices in my home network, it did not work for the opendtu. I had to replace the name of the power meter by its local ip address. Then it worked.
